docs/evaluation/Dummy/document.html

```diff
@@ -0,0 +1,441 @@
+<!DOCTYPE html>
```

Comment:
This file is not a "dummy" anymore, please take a look at the muncher dummy, where the HTML file is just created in the Dockerfile: https://github.com/o2r-project/o2r-muncher/blob/master/test/workspace/dummy/Dockerfile

Other things:

1. rename the `Dummy` directory to `dummy-workspace`